pkg/qemu/qemu.go

```diff
@@ -135,6 +138,10 @@ func (o *Options) Cmdline() ([]string, error) {
 		args = append(args, "-initrd", o.Initramfs)
 	}
 
+	if len(o.SMBIOSArgs) != 0 {
```

Comment:
The way to add arbitrary args would be to use a qemu.Device like below

Reply:
Ah - I see. I can do this - however, the naming `Devices` is a bit misleading. But let me try it through the Devices Option then. Thanks!